apps/coscrad-frontend/src/components/resources/utils/render-audio-preview.tsx

```diff
@@ -0,0 +1,14 @@
+import { PlayArrowRounded } from '@mui/icons-material';
+import { Button } from '@mui/material';
+import { CellRenderer } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types';
+
+export interface HasUrl {
+    url: string;
+}
+
+export const renderAggregateUrlCell: CellRenderer<HasUrl> = ({ url }: HasUrl) => (
```

Comment:
This may be helpful in the playlist **detail** view. We'll have to see how we can leverage this over there.

Reply:
done